patches/react-native-safe-area-context+4.7.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
+index ca5646c..7bcc64d 100644
+--- a/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
++++ b/node_modules/react-native-safe-area-context/android/src/main/java/com/th3rdwave/safeareacontext/SafeAreaView.kt
+@@ -13,7 +13,7 @@ import java.util.*
+ import java.util.concurrent.locks.ReentrantLock
+ import kotlin.concurrent.withLock
+ 
+-private const val MAX_WAIT_TIME_NANO = 500000000L // 500ms
++private const val MAX_WAIT_TIME_NANO = 5000000L // 5ms
```

Comment:
Just curiosity, why are we reducing this time?

Reply:
With the introduction of RN 72, the main thread was being blocked for half a second causing the navigation to be absolutely terrible, this change makes it smooth and causes no issues whatsoever. In fact if you set this to 10s, the new screen won't display for 10s as there is a lock using this value